mgwr/gwr.py

```diff
@@ -335,11 +340,7 @@ def fit(self, ini_params=None, tol=1.0e-5, max_iter=20, solve='iwls',
             else:
                 m = self.points.shape[0]
 
-            if pool:
-                rslt = pool.map(self._local_fit,
-                                range(m))  #parallel using mp.Pool
-            else:
-                rslt = map(self._local_fit, range(m))  #sequential
+            rslt = Parallel(n_jobs=self.n_jobs)(delayed(self._local_fit)(i) for i in range(m))
```

Comment:
> Would make sense to keep this if/else structure to allow running single-core outside of `Parallel`? It tends to produce more understandable Traceback in case something goes wrong.

My understanding is that when n_jobs=1, it will default to the [normal traceback](https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html)? I can check.

```
Easier debugging:
informative tracebacks even when the error happens on the client side
using ‘n_jobs=1’ enables to turn off parallel computing for debugging without changing the codepath
early capture of pickling errors
```

Reply:
Ah, cool. Probably some ancient or completely wrong information sits in my head. Sorry for confusion!